### ISSUE TITLE
Deprecation warnings from dependencies?

### ISSUE THREAD

**Issue by ditman** [lewagon/wait-on-check-action#76]
Hello team!

We're using lewagon/wait-on-check-action, and we've started seeing deprecation warnings coming from the dependencies of the action in [our logs](https://github.com/flutter/plugins/actions/runs/3542087063):

**ruby/setup-ruby@1.81.0**

```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using
Environment Files. For more information see:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**actions/cache@v2**

```
Warning: The `save-state` command is deprecated and will be disabled soon. Please upgrade to using
Environment Files. For more information see:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

Also a warning about "Node.js 12 actions being deprecated" for both of the above:

```
Node.js 12 actions are deprecated. For more information see:
https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/.
Please update the following actions to use Node.js 16: ruby/setup-ruby@v1.81.0, actions/cache@v2
```

**Is this something that we can fix on our end, or is this something that the `wait-on-check-action` needs to fix?**

Thanks!

**Comment by quinnturner** [comment]
This will be solved with my PRs here: #74 #75.

**Comment by ditman** [comment]
Thanks for the fixes @quinnturner!
